var/spack/repos/builtin/packages/py-evodiff/package.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyEvodiff(PythonPackage):
+    """Python package for generation of protein sequences and evolutionary alignments via \
+        discrete diffusion models"""
+
+    homepage = "https://github.com/microsoft/evodiff"
+    pypi = "evodiff/evodiff-1.1.0.tar.gz"
+
+    license("MIT", checked_by="ashim-mahara")
+
+    version("1.1.0", sha256="c1f2d7bd0e46ad244f1c55066caefc5ad9b1bcf4e836be1832311b8cd74e923f")
```

Comment:
Missing `py-mlflow` dependency

Reply:
https://github.com/microsoft/evodiff/pull/44

Reply:
they removed the dependency, i couldn't find it in the main branch

Reply:
@meyersbs

Reply:
Don't look at the branch, look at the .tar.gz from the release: https://files.pythonhosted.org/packages/52/9a/9d4ee39c6d6fbc179b36c36db80b4bf37c8dc91f3fad38a3736472c1ce1f/evodiff-1.1.0.tar.gz

```
    install_requires=[
        "pandas",
        "lmdb",
        "numpy",
        "sequence-models",
        "mlflow",
        "scikit-learn",
        "blosum",
        "seaborn",
        "matplotlib",
        "fair-esm",
        "tqdm",
        "biotite",
        "requests",
        "mdanalysis",
        "pdb-tools"
    ],
```

Reply:
done